package.json

```diff
@@ -28,8 +28,8 @@
   },
   "homepage": "https://github.com/ionic-team/ionic-pwa-elements",
   "devDependencies": {
-    "@stencil/core": "^1.8.11",
-    "np": "^6.5.0",
-    "workbox-build": "^4.3.1"
+    "@stencil/core": "^3.4.0",
+    "np": "^8.0.4",
```

Comment:
FW-4065 says @types/node is needed as a dev dependency. Is this still the case?

Reply:
I ran into the same error reported in FW-4065, but upgrading the `np` and `workbox-build` dependency resolved the issue.